classes/person.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+att_accessor :name, :age
+att_reader :id
+
```

Comment:
- [ ] As requested by the previous reviewer, These attributes still have typos, it's supposed to be `attr_` instead of `att_`, and attributes are supposed to be at the beginning of the class, not outside it. Please fix that.
```suggestion
class Person < Nameable
  att_accessor :name, :age
  att_reader :id
   ----
```